src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -188,6 +262,9 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
                 this.props.onLoaded(this.videoPlayer.current.video.video);
             }
         });
+
+        // Once the video is loaded, add any asset timeline tags
```

Comment:
`onVideoStateChange` fires a lot... why do we need to keep re-rendering the tick marks?  Doesn't it only change if the child assets change?

Reply:
I'm not sure I understand this question? We only add the timeline marks on load and when the child assets change, it's never called in onVideoStateChange?

Reply:
Sorry, the github UI confused me 👍